packages/ui/src/memberships/components/MemberProfile/MemberAccounts.tsx

```diff
@@ -6,35 +6,70 @@ import { AccountRow } from '../../../common/components/Modal'
 import { RowGapBlock } from '../../../common/components/page/PageContent'
 import { SidePaneLabel } from '../../../common/components/SidePane'
 import { Member } from '../../types'
+import { TokenValue } from '@/common/components/typography'
+import { isDefined } from '@/common/utils'
+import { useBalance } from '@/accounts/hooks/useBalance'
 
-export const MemberAccounts = ({ member }: { member: Member }) => (
-  <AccountsDisplay gap={16}>
-    <SidePaneLabel text="Root account" />
-    {!!member.rootAccount && (
-      <AccountRow>
-        <UnknownAccountInfo address={member.rootAccount} placeholderName="Root Account" />
-      </AccountRow>
-    )}
-    <SidePaneLabel text="Controller account" />
-    {!!member.controllerAccount && (
-      <AccountRow>
-        <UnknownAccountInfo address={member.controllerAccount} placeholderName="Controller Account" />
-      </AccountRow>
-    )}
+export const MemberAccounts = ({ member }: { member: Member }) => {
+  const rootBalance = useBalance(member.rootAccount);
+  const countrolBalance = useBalance(member.controllerAccount);
 
-    {!!member.boundAccounts.length && (
-      <>
-        <SidePaneLabel text="Bound accounts" />
-        {member.boundAccounts.map((account) => (
-          <AccountRow key={account}>
-            <UnknownAccountInfo address={account} placeholderName="Bound Account" />
-          </AccountRow>
-        ))}
-      </>
-    )}
-  </AccountsDisplay>
-)
+  return (
+    < AccountsDisplay gap={16} >
+      <SidePaneLabel text="Root account" />
+      {!!member.rootAccount && (
+        <AccountMemberRow>
+          <UnknownAccountInfo address={member.rootAccount} placeholderName="Root Account" />
+          <TokenValue value={rootBalance?.total} isLoading={!isDefined(rootBalance?.total)} />
```

Comment:
What could be the reason that it keeps blinking (never is defined)?

![blinking](https://github.com/Joystream/pioneer/assets/31551045/7420641f-8e36-4240-b3b6-38e809fa035e)